wiki/quest-modding.md

```diff
@@ -1,14 +1,58 @@
 # Quest Modding
-_A quick summary of Quest Modding_
 
-## Quest Mods
-::: warning
-This page is a WIP and will be updated with time
-:::
+## Installation 
+
+Currently there are two working tools for installing custom songs and mods on your Quest:
+* [BeatSaberQuestInstaller (windows only GUI for installing BMBF)](https://github.com/NyanBlade/BeatSaberQuestInstaller/releases)
+* [BMBF installed via SideQuest](https://github.com/kihecido/BMBF/releases)
+
+For new users, BeatSaberQuestInstaller may be easier to use as it handles some of the more confusing parts of modding/updating a modded game, such as restoring scores and installing adb.
+
+### Installing with BeatSaberQuestInstaller
+
+Before starting the process go to the utilities tab and backup your scores, if you've previously modded Beat Saber you also need to go there to uninstall the game, make sure you backup **BEFORE** uninstalling.
+
+![BeatSaberQuestInstaller](./images/beginners-guide/qbsioptions.png)
+
+Once you've done what you need, if you're a new user follow all the steps from step 1 and if you're a returning user skip to step 4 in the installer, after completing the steps open BMBF from the channels tab in Oculus TV or Unknown Sources in your library.
+
+### Installing BMBF with SideQuest
+
+If you haven't already, download and setup [SideQuest](https://sidequestvr.com/#/setup-howto)
+
+Open SideQuest and connect your Quest to your PC. If you've previously modded Beat Saber or have scores you want to backup, go to `My Apps` located in the top bar of the window and find Beat Saber.
+
+Click the cog next to it then press the `BACKUP GAME DATA` button, if you have a modded game you also need to uninstall it by pressing the `UNINSTALL APP` button.
+
+![SideQuestUninstall](./images/beginners-guide/sqbackupanduninstall.png)
```

Comment:
While not essential, consider darkening the unimportant parts of this image, as I did in my example image above.